server/state/state_store.go

```diff
@@ -0,0 +1,29 @@
+package state
+
+import (
+	"github.com/kubernetes-incubator/cri-o/oci"
+	"github.com/kubernetes-incubator/cri-o/server/sandbox"
+)
+
+// Store stores the state of the CRI-O server, including active pods and containers
+type Store interface {
```

Comment:
do you mind if we don't add another interface with the name `\<Store\>`? maybe `StateStorer` or something more unique to grep for?